__tests__/integration/utils/wallet-helper.js

```diff
@@ -0,0 +1,174 @@
+import { loggers } from '../txLogger';
+import { TestUtils } from './test-utils-integration';
+
+/**
+ * A helper for testing the wallet
+ */
+export class WalletHelper {
+  /**
+   * Identification that will be used to start the wallet and as a reference on the request headers
+   */
+  #walletId;
```

Comment:
What is this `#` used for?

Reply:
This `#` is the syntax for [private class features](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields), to better encapsulate the wallet data.